src/components/Navigator/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import TriangleIcon from "@/assets/TriangleIcon.svg"
+import style from "./index.module.scss";
+
+const Navigator = () => {
+  const navLinks = [
+    {
+      name: "HOME",
+    },
+    {
+      name: "SPEAKERS",
+      child: [],
+    },
+    {
+      name: "SCHEDUAL",
+      child: [],
+    },
+    {
+      name: "CONTACT US",
+    },
+    {
```

Comment:
Subscribe should be a button not a navlink. The Gap between Navlinks is 40px, but subscribe button left margin is 50px.

Reply:
Thank you！I have fix this problem
![WechatIMG334](https://github.com/taodemy/taotify-website/assets/89081041/eefeafad-ae4c-4ff0-999a-b86a50801eb0)